api/controllers/delegates.js

```diff
@@ -101,4 +104,37 @@ DelegatesController.getForgers = function (context, next) {
 	});
 };
 
+DelegatesController.getForgedByAccount = function (context, next) {
+	var params = context.request.swagger.params;
+	
+	var filters = {
+		generatorPublicKey: params.publicKey.value,
+		start: params.fromTimestamp.value || constants.epochTime.getTime(),
+		end: params.toTimestamp.value || Date.now()
+	};
+
+	modules.blocks.utils.aggregateBlocksReward(filters, function (err, reward) {
+		if (err) { 
+			if (err === 'Account not found or is not a delegate') {
+				return next(swaggerHelper.generateParamsErrorObject([params.publicKey], ['Account not found or is not a delegate']));
+			} else {
+				return next(err); 
+			}
+		}
+
+		var forged = new bignum(reward.fees).plus(new bignum(reward.rewards)).toString();
+		var response = {
+			data: {
+				fees: reward.fees,
+				rewards: reward.rewards,
+				forged: forged,
+				count: reward.count
+			},
+			meta: {},
```

Comment:
Since we are using some dynamic default values for `start` and `end`. Add these information to `meta` object. So user should be aware the time line on which these stats belongs to if he don't provided the param himself.